azext_iot/tests/central/json/device_twin.json

```diff
@@ -1,186 +1,76 @@
                     "processorArchitecture": {
-                        "$lastUpdated": "current_time",
-                        "value": {
-                            "$lastUpdated": "current_time"
-                        }
-                    },
-                    "swVersion": {
-                        "$lastUpdated": "current_time",
-                        "value": {
-                            "$lastUpdated": "current_time"
-                        }
+                        "$lastUpdated": "current_time"
                     },
                     "processorManufacturer": {
-                        "$lastUpdated": "current_time",
-                        "value": {
-                            "$lastUpdated": "current_time"
-                        }
+                        "$lastUpdated": "current_time"
                     },
-                    "totalStorage": {
-                        "$lastUpdated": "current_time",
-                        "value": {
-                            "$lastUpdated": "current_time"
-                        }
+                    "swVersion": {
+                        "$lastUpdated": "current_time"
                     },
                     "totalMemory": {
-                        "$lastUpdated": "current_time",
-                        "value": {
-                            "$lastUpdated": "current_time"
-                        }
-                    }
-                },
-                "$iotin:settings": {
-                    "$lastUpdated": "current_time",
-                    "fanSpeed": {
-                        "$lastUpdated": "current_time",
-                        "value": {
-                            "$lastUpdated": "current_time"
-                        },
-                        "sc": {
-                            "$lastUpdated": "current_time"
-                        },
-                        "sd": {
-                            "$lastUpdated": "current_time"
-                        },
-                        "sv": {
-                            "$lastUpdated": "current_time"
-                        }
+                        "$lastUpdated": "current_time"
+                    },
+                    "totalStorage": {
+                        "$lastUpdated": "current_time"
                     }
                 }
             },
-            "$version": 2728
+            "$version": 2,
+            "device_info": {
+                "__t": "c",
+                "manufacturer": "Xiaomi",
+                "model": "Redmi Note 5",
+                "osName": "Android",
+                "processorArchitecture": "I386",
+                "processorManufacturer": "ILOnly",
+                "swVersion": "9",
+                "totalMemory": 3741,
+                "totalStorage": 54621
+            }
         }
     },
-    "capabilities": {
-        "iotEdge": "False"
+    "status": "enabled",
+    "statusUpdateTime": "0001-01-01T00:00:00Z",
+    "tags": {},
```

Comment:
add _links as a property and in the test make sure that the _links are not shown. Also make sure tags are shown in the test

Reply:
added